examples/plot_example_workflow.py

```diff
@@ -0,0 +1,148 @@
+"""
+===============================
+Example ``autoreject`` workflow
```

Comment:
I would change the title to be something about ICA and autoreject.

All the examples are detailing some part of the autoreject workflow